packages/dara-core/dara/core/persistence.py

```diff
@@ -281,7 +281,8 @@ async def write(self, value: Any, notify=True):
         """
         Persist a value to the store.
 
-        If scope='user', the value is written for the current user.
+        If scope='user', the value is written for the current user so the method can only
+        be used in authenticated contexts.
```

Comment:
Do we have docs to point to here for authenticated contexts? I think that is something that will be confusing to people

Reply:
We don't have a dedicated page but it is mentioned a couple times. Perhaps deserves a separate authentication related page (I think there was an issue raised on GH asking for that) but I think that's a separate effort, will try to remember to link it here once we get around to it

Reply:
Fair, could even be on the FAQ depending on how much we want to say there